store/gcworker/gc_worker.go

```diff
@@ -309,13 +309,32 @@ func (w *GCWorker) leaderTick(ctx context.Context) error {
 		return nil
 	}
 
+	concurrency, err := w.getGCConcurrency(ctx)
+	if err != nil {
+		logutil.Logger(ctx).Info("[gc worker] failed to get gc concurrency.",
+			zap.String("uuid", w.uuid),
+			zap.Error(err))
+		return errors.Trace(err)
+	}
+
 	ok, safePoint, err := w.prepare(ctx)
-	if err != nil || !ok {
-		if err != nil {
-			metrics.GCJobFailureCounter.WithLabelValues("prepare").Inc()
-		}
+	if err != nil {
+		metrics.GCJobFailureCounter.WithLabelValues("prepare").Inc()
 		return errors.Trace(err)
+	} else if !ok {
```

Comment:
```suggestion
	} else if !ok && w.logBackupEnabled {
```

Reply:
> 

Whether w.logBackupEnable is true or false, It need return at line:334

Reply:
OK